spec/models/company_spec.rb

```diff
@@ -30,6 +30,55 @@
   let(:payment_date_2018) { Time.zone.local(2018, 11, 21) }
   let(:payment_date_2020) { Time.zone.local(2020, 3, 15) }
 
+  let(:company_emp_cats) { create(:company) }
```

Comment:
These should be created within the example group where they're used. They're not used by most examples.

Reply:
Done.